### ISSUE TITLE
Pypi version has odd need for argument labels

### ISSUE THREAD

**Issue by bwrsandman** [glenfletcher/Equation#29]
This is only on the Pypi version 1.1.01

The following gives an error

``` python
Expression("x")(1)
TypeError: <Equation.core.Expression(x) object at 0x7fc39a888590>() takes at least 1 arguments (1 given) 'x' not defined
```

but the following is ok

``` python
Expression("x")(x=1)
1
```

The error does not seem to happen on maste, it might be a good idea to add unittest for that to avoid future regressions.

**Comment by glenfletcher** [comment]
This Works in v1.2, however related error of 

``` python
Expression("x+x-y")(1,2)
TypeError: <Equation.core.Expression(((x + x) - y)) object at 0x0202bed0>() takes at least 2 arguments (2 given) 'y' not defined
```

while

``` python
Expression("x+x-y")(1,y=2)
0
```

works

**Comment by glenfletcher** [comment]
Fixed in Bug-Fix#29, v1.2 to be released today, won't fix v1.1.x
